.github/workflows/terraform-apply.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
           terraform apply -var "channel=${{ inputs.channel }}" -var "model_name=${{ inputs.model }}" --auto-approve
       - name: Wait for the application to be active or blocked
         run: |
-          juju wait-for model ${{ inputs.model }} --query='forEach(applications, app => (app.status == "active" || app.status == "blocked") )'
+          juju wait-for model ${{ inputs.model }} --query="forEach(units, unit => (unit.workload-status == 'active' || unit.workload-status == 'blocked'))"
```

Comment:
suggestion: add a comment with link to issue stating we need to check on the unit status not application status, to make sure it doesn't get changed for any reason unless the issue is resolved.

Reply:
+1 added